messaging/artemis-jta/src/test/java/io/quarkus/ts/openshift/messaging/artemisjta/AbstractArtemisTest.java

```diff
@@ -0,0 +1,117 @@
+package io.quarkus.ts.openshift.messaging.artemisjta;
+
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+import static io.restassured.RestAssured.given;
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public abstract class AbstractArtemisTest {
+    @Test
+    @Order(1)
+    public void testPrice() {
+        given().queryParam("fail", "false").body("666")
+                .when()
+                .post("/price").then().statusCode(200);
+
+        await().atMost(10, TimeUnit.SECONDS).untilAsserted(() -> assertThat("testPrice:",
```

Comment:
RestAssured can do asserts on its own, you don't have to do the `assertThat(... extract().body().asString(), equalTo(...))` dance. Something like:

```java
given()
.when()
    .get("/price")
.then()
    .statusCode(200)
    .body(equalTo("666:666"));
```